.github/workflows/validate.yml

```
@@ -16,19 +16,17 @@ jobs:
     if: ${{ !contains(github.head_ref, 'all-contributors') }}
     strategy:
       matrix:
-        node: [10.18, 10, 12, 14, 15]
+        node: [10.18, 10, 12, 14, 16]
```

Node 15 is still supported until June 1
```suggestion
        node: [10.18, 10, 12, 14, 15, 16]
```

---

We shouldn't support odd Node versions, only when they're `latest`

---

As far as I understand from [Node's LTS schedule](https://github.com/nodejs/Release), 15 is actually still supported right now, even though 16 (which will eventually be upgraded to an LTS) has been released since. Technically 16 is the only latest release here anyway.

---

15 is officially EOL now.